client/context.go

```diff
@@ -308,6 +308,15 @@ func (ctx Context) WithPreprocessTxHook(preprocessFn PreprocessTxFn) Context {
 	return ctx
 }
 
+// GetCmdContextOr returns the CmdContext if it is not nil, otherwise it returns the
+// provided context.
+func (ctx Context) GetCmdContextOr(goctx context.Context) context.Context {
```

Comment:
```suggestion
func (ctx Context) GetCmdContextWithFallback(goctx context.Context) context.Context {
```

Reply:
Is there ever a case where we would use `ctx.GetCmdContextOr()` and not provide `context.Background()` as the fallback?  Seems like we could just simplify and always return a context.Background()